player.js

```diff
@@ -0,0 +1,58 @@
+Scrabble = require('./scrabble');
+var scrabble = new Scrabble();
+
+var Player = function(name){
+  this.plays = [];
+  this.name  = name;
+
+  this.play = function(word) {
+    if (this.hasWon === true) {
+      return false;
+    }
+    this.plays.push(word);
+    return this.plays;
+  };
+
+  this.total_score = function() {
+    var sum = 0;
+    for(var word of this.plays){
+      sum = sum + scrabble.score(word);
+    }
+    return sum;
+  };
+
+  this.hasWon = function() {
+    if (this.total_score() > 100 ){
+      return true;
+    }
+    return false;
+  };
+
+  this.highestScoringWord = function() {
+    var max = 0,
+        word_best = "";
+    for(var word of this.plays){
+      if (scrabble.score(word) > max) {
+        max = scrabble.score(word);
+        word_best = word;
+      }
+    }
+    return word_best;
+  };
```

Comment:
this is another function where you could store a variable for the currebt word_best + its score, then check each word played against the current word_best, instead of having to calculate all words' scores every time